<launch>
    <node name="uav_pid_control_node" pkg="uav_control" type="uav_pid_control_node" output="screen">
      <rosparam command="load" file="$(find uav_control)/config/pid_controller.yaml" />
      <rosparam command="load" file="$(find uav_control)/config/vehicle.yaml" />
    </node>

</launch>